modules/hooks.nix

```diff
@@ -96,5 +104,11 @@ in {
           entry = "${tools.terraform-fmt}/bin/terraform-fmt";
           files = "\\.tf$";
         };
+      yamllint =
+        {
+          name = "yamllint";
+          description = "Yaml linter";
+          entry = "${tools.yamllint}/bin/yamllint";
```

Comment:
This one seems generic enough to use a pattern that matches all yaml files. Upstream also [does this](https://github.com/adrienverge/yamllint/blob/master/.pre-commit-hooks.yaml).

Reply:
That's the reason why I didn't specify the 'files' properties.
Is there any left to do to improve the PR ?
Thx for the review.

Reply:
Ah so I was expecting something like `types = ["file" "yaml"]` here.
After enabling, users can still opt out for some set of files via the `excludes` options.

Reply:
Apply in c28ce2cd8a8ce1b4e2761f61da10dd79cbd5b8aa.
I have to admit I don't quite get the syntax ... Are the two elements of the list treated as key/value ?

Reply:
It's what pre-commit does. Apparently it uses [this mapping](https://github.com/chriskuehl/identify/blob/master/identify/extensions.py). How [types](https://pre-commit.com/#filtering-files-with-types) are combined is left implicit in the docs, but it needs to be set intersection / logical and to make sense.